core/src/main/java/org/elasticsearch/action/support/ToXContentToBytes.java

```diff
@@ -75,6 +76,8 @@ public final String toString() {
             toXContent(builder, EMPTY_PARAMS);
             return builder.string();
         } catch (Exception e) {
+            // So we have a stack trace logged somewhere
+            ESLoggerFactory.getLogger(getClass().getName()).warn("Error building toString", e);;
```

Comment:
Does or doesn`t? Not quiet sure from your comment if you think this is potentially harmfull or not.